.github/workflows/doc.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024 Adevinta
+
+name: Documentation
+on:
+  push:
+    branches:
+    - mdBook
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+jobs:
+  build:
+    name: Generate documentation
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Build documentation
+        run: ./build.bash
+        env:
+          LAVA_VERSION: latest
```

Comment:
I think we should always build the documentation for the latest version of Lava. At the end of the day we are only providing documentation for the latest release.